dagger-functions/build.gradle

```diff
@@ -53,7 +53,7 @@ dependencies {
     compileOnly group: 'org.apache.flink', name: 'flink-table-api-java-bridge_2.11', version: flinkVersion
     compileOnly group: 'org.apache.flink', name: 'flink-streaming-java_2.11', version: flinkVersion
     compileOnly group: 'org.apache.flink', name: 'flink-metrics-dropwizard', version: flinkVersion
-    compileOnly group: 'com.gojek', name: 'stencil', version: '2.0.15'
+    compileOnly group: 'io.odpf', name: 'stencil', version: '0.1.6'
```

Comment:
we can remove this now, as we are already dependent upon common

Reply:
it's showing compilation error If I remove it. I guess commomDependentjar is not getting added to compileclass path in dagger-functions.